common/main/ka.xml

```diff
@@ -6607,6 +6607,11 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<ordinalMinimalPairs ordinal="many">მე-{0}</ordinalMinimalPairs>
 			<ordinalMinimalPairs ordinal="one">{0}-ლი</ordinalMinimalPairs>
 			<ordinalMinimalPairs ordinal="other">{0}-ე</ordinalMinimalPairs>
+			<caseMinimalPairs case="nominative">{0} ಮರವು ದೊಡ್ಡದಾಗಿದ</caseMinimalPairs>
```

Comment:
Where did the Georgian values come from? I couldn't find them in the review sheet.

Reply:
My fault, will fix! that was supposed to be an addition for Kannada.